src/Dapr.Actors/Runtime/ActorRuntimeOptions.cs

```diff
@@ -119,6 +121,25 @@ public bool DrainRebalancedActors
             }
         }
 
+        /// <summary>
+        /// A configuration that defines the Actor Reentrancy parameters. If set and enabled, Actors
+        /// will be able to perform reentrant calls to themselves/others. If not set or false, Actors
+        /// will continue to lock for every request.
+        /// See https://docs.dapr.io/developing-applications/building-blocks/actors/actor-reentrancy/
+        /// </summary>
+        public ActorReentrancyConfig ReentrancyConfig
```

Comment:
IMO this is should be initialized to a default value so you can do `options.ReentrancyConfig.Foo = bar`.